docs/content/components/footer.mdx

```diff
@@ -85,7 +85,7 @@ code_snippets:
           class="usa-footer usa-footer--nci-big"
           id="nci-footer"
           aria-label="Footer">
-          <div class="grid-container usa-footer__return-to-top">
+          <div class="grid-container usa-footer__nci-return-to-top">
```

Comment:
Default, the html should be exactly the same as the html in the component page. 

```html
<div class="grid-container usa-footer__return-to-top">
  <a href="#top" aria-label="Back To Top"><span>Back To Top</span></a>
</div>
```

When javascript is loaded, or when react renders the component, it needs to be the exact same as what ncids-js does, which changes the html to

```html
<div class="usa-footer__nci-return-to-top ['show' or 'hide' class here depending on state]">
    <a href="#top" aria-label="Back To Top"><span>Back To Top</span></a>
</div>
```